reference/plugins.rst

```diff
@@ -0,0 +1,282 @@
+.. _plugins_reference:
+
+Plugins
+=======
+
+.. warning::
+
+    This is an EXPERIMENTAL feature released in 1.8. Subject to breaking changes.
+
+The Conan plugins are Python functions that are intended to extend the Conan functionalities and let users customize the client behavior at
+determined execution points.
+
+Storage, activation and sharing
```

Comment:
I think this section or the information contained. shouldn't be in the reference but the extending/plugins.